corehq/apps/sso/templates/sso/config_errors.html

```diff
@@ -15,7 +15,7 @@ <h2>
               {% endblocktrans %}
             </h2>
 
-            {% if error.code == error.REDIRECT_INVALID_URL %}
+            {% if is_saml and error.code == error.REDIRECT_INVALID_URL %}
```

Comment:
no, I don't think it will simplify things. this template is rendered from multiple views and the saml error codes are generated from the OneLogin SAML library

Also, I always feel user facing text should be in templates over backend code.